src/pages/on-chain-governance/shared/getAllCycleVotes.ts

```diff
@@ -0,0 +1,15 @@
+import { ListVotesResult } from "@defichain/jellyfish-api-core/dist/category/governance";
+
+export function getAllCycleVotes(allCycleProposalVotes: ListVotesResult[]): {} {
```

Comment:
Normally if we refactor into a standalone file it is because it has use case in more than one file. Does it apply to this function?